### DISCUSSION TITLE
Changing Timezone in SOC

### DISCUSSION THREAD

**Discussion by cyberminded** [Security-Onion-Solutions/securityonion#3764] [Unsupported Versions] [answered]
Hello,

When I run timedatectl in Terminal, it says that the local time for my Security Onion vm is set to UTC. However, the time zone in the Security Onion Console is set to Africa/Abidjan. Is this tied to my browser somehow? Any idea how I would get this set to UTC?

**Comment by cyberkimh** [comment]
Yes we are having this issue too. Any help would be appreciated.

**Reply by marianoka** [reply]
hi!
to change timezone do the following steps
1- rm /etc/localtime
2- sudo ln -s /usr/share/zoneinfo/America/yourcity /etc/localtime
3-timedatectl to check

**Comment by Stephen-Ridgway** [comment]
Just a quick update: we have an instance of a master server where the console shows the timezone as Africa/Abidjan regardless of the OS timezone (which is correctly set to UTC). Could this be a Kibana bug similar to https://github.com/elastic/kibana/issues/57457?

Manually setting Kibana's timezone to UTC did not fix the problem.

**Comment by jertel** [comment, marked as answer]
Yes, the browser's timezone is picked up by SOC and used when showing dates and times. The method used to pick the browser's timezone is appropriately named `tz.guess()`, as it isn't perfect but usually gets it right. There's currently no setting in SOC to adjust this default timezone. However, we're aware of the desire to change this in some situations and tracking in #4261.
